plugin/trino-ai-functions/src/main/java/io/trino/plugin/ai/functions/AiProvider.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ai.functions;
+
+public enum AiProvider
+{
+    OLLAMA,
```

Comment:
Let's rather use strings instead of enums.
There are more AI services out there and limiting the providers to only the ones we know of seems a limiting factor.

I'd salute having a more pluggable way to think about AI providers.

Reply:
We don't support plugins for plugins in Trino. Adding new providers will require modifying the code, so anyone doing that can easily add their own enum.

Using an enum means that users will get an immediate config validation error if the value doesn't exist. I've been changing other similar string configs to enums.

Reply:
Also .. I am sure we are happy to expand to other providers but with Ollama we support a LOT already